src/components/ColorPicker/ColorPicker.tsx

```diff
@@ -41,25 +51,40 @@ export class ColorPicker extends React.PureComponent<ColorPickerProps, State> {
       return;
     }
 
-    this.setState({pickerSize: colorNode.clientWidth});
+    this.setState({
+      pickerWidth: colorNode.clientWidth,
+      pickerHeight: colorNode.clientHeight,
+    });
 
     if (process.env.NODE_ENV === 'development') {
       setTimeout(() => {
-        this.setState({pickerSize: colorNode.clientWidth});
+        this.setState({
+          pickerWidth: colorNode.clientWidth,
+          pickerHeight: colorNode.clientHeight,
+        });
       }, 0);
     }
   }
 
   render() {
     const {id, color, allowAlpha} = this.props;
     const {hue, saturation, brightness, alpha: providedAlpha} = color;
-    const {pickerSize} = this.state;
+    const {pickerWidth, pickerHeight} = this.state;
```

Comment:
Just curious, do we need to update these values on resize?

Reply:
Good point, they should be updated 👍